<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #30343d;
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="logobar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="menu" native="true">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: #30343d;   /* color de fondo plano */
  color: #FFFFFF;              /* texto blanco */
  border: none;                /* sin borde */
  padding: 8px 12px;           /* espaciado interno */
  border-radius: 4px;          /* esquinas redondeadas */
  text-align: left;            /* icono + texto alineados a la izquierda */
}
QPushButton:hover {
  background-color: #585e6c;   /* ligero cambio al pasar el rat칩n */
}
QPushButton:pressed {
  background-color: #1f1f1f;   /* tono m치s oscuro al pulsar */
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="ButtonVerMaterias">
            <property name="text">
             <string>Ver Materias</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/arrow-right-circle.svg</normaloff>:/icons/icons/arrow-right-circle.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonCalcularSemestres">
            <property name="text">
             <string>Calcular Semestres</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/grid.svg</normaloff>:/icons/icons/grid.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonAgregarPrerrequisito">
            <property name="text">
             <string>A침adir Prerrequisitos</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/plus.svg</normaloff>:/icons/icons/plus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonAgregarMateria">
            <property name="text">
             <string>A침adir Materias</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/plus.svg</normaloff>:/icons/icons/plus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonEliminarMateria">
            <property name="text">
             <string>Eliminar Materias</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/minus.svg</normaloff>:/icons/icons/minus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonEliminarPrerrequisito">
            <property name="text">
             <string>Eliminar Prerequisito</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/minus.svg</normaloff>:/icons/icons/minus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonCargar">
            <property name="text">
             <string>Cargar</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/upload.svg</normaloff>:/icons/icons/upload.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonGuardar">
            <property name="text">
             <string>Guardar</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/save.svg</normaloff>:/icons/icons/save.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonNuevo">
            <property name="text">
             <string>Nuevo</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/file-plus.svg</normaloff>:/icons/icons/file-plus.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="panel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #d5d8df;
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
